extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -145,7 +150,11 @@ export class DbPanel extends DisposableObject {
 
     const owner = getOwnerFromGitHubUrl(ownerName) || ownerName;
     if (!isValidGitHubOwner(owner)) {
-      throw new Error(`Invalid user or organization: ${owner}`);
+      void showAndLogErrorMessage(`Invalid user or organization: ${owner}`);
+    }
+
+    if (this.dbManager.doesRemoteOwnerExist(owner)) {
+      void showAndLogErrorMessage(`The owner '${owner}' already exists`);
```

Comment:
From testing: when I add a duplicate (for repos/owners/lists), I now get two error messages 🙈 

![image](https://user-images.githubusercontent.com/42641846/208903944-f4d0180f-f722-4846-b837-1fb8c87e508f.png)

Can we only show one of them?

Reply:
🤦 I forgot to `return` after showing notifications.